### PR TITLE
Remove openblas version specifier

### PR DESCRIPTION
**Pull request by guyer**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by guyer** [comment]
@conda-forge-admin, please rerender

**Comment by guyer** [comment]
@conda-forge-admin, please rerender

**Comment by guyer** [comment]
@conda-forge/core what does "I tried to {} for you but ran into some issues" mean?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
> @conda-forge/core what does "I tried to {} for you but ran into some issues" mean?

I was able to lint and re-render locally so I guess that is a problem with the webservices. Probably safe to ignore if you want to merge this.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,18 +20,19 @@ source:
...
-  number: 5
+  number: 6
   skip: True  # [not py27]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - mkl-devel  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: This should be in `host`

**Comment by guyer** [comment]
OK, thanks @ocefpaf. 
Now I'm fighting Windows build  errors that weren't there a few hours earlier. I hate conda-forge so much.

**Comment by isuruf** [comment]
Defaults' `mkl` 2018 had the import libs, but 2019 doesn't and they were split off into `mkl-devel`. (It's not conda-forge's fault that defaults channel does breaking changes)

**Comment by guyer** [comment]
@isuruf  Thanks for the explanation. I'd guessed that `mkl-devel` was what was needed, based on gmsh-feedstock.

I do appreciate that conda-forge is not conda. I hope you appreciate that if the whole thing is an unreliable house of cards, I don't really care where the specific fault lies.

**Comment by guyer** [comment]
@isuruf & @ocefpaf travis-ci builds are taking hours before they even start, apparently because of throttling due to everything else in conda-forge rebuilding all the time. Can I safely merge if the azure osx build is successful?

**Pull request merged**